GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+
+    * Create a Cobalt admin service-principal/service-endpoint for Cobalt deployments
+        * Sign-in to your organization's Azure account. (https://portal.azure.com)
+        * Filter for Azure Active Directory and navigate to it's menu
+        * Select App registrations from the menu blade
+        * Click [Add/+] New registration then enter a name for the application (ex. cobalt-hw-admin-sp)
+        * Choose single tenant as a supported account type to keep things simple.
+        * Click Register
+
+    * Configure the newly created registered app as a Service Principal
+        * From the app's service blade, select the Certificates & Secrets tab
+        * Click [Add/+] New from within Client secrets menu then enter a description (ex. rbac)
+        * Click Add
+            * NOTE: Take note of the generated client secret (only displayed once).
+        * From the app's service blade, select Overview. Take note of the Application (client) ID for the next step.
+
+    * Grant newly created Service Principal a Contributor role to your preferred enterprise subscription
+        * Filter for subscriptions and navigate to the subscriptions list
+        * Either choose a subscription or create a new one
+        * Select your chosen subscription then select the Access control (IAM) tab from the menu blade.
+        * Click [+/Add] and select Add role assignment
+            * From the sub-menu, select 'Contributor' as a role from the drop down and search for the newly created Service Principal (i.e. cobalt-hw-admin-sp)
+            * Click Save
+            * ![image](.role-assignment-menu.png)
+
+    * Create Resource Group and Storage Account for backend state
+        * Filter for Storage accounts and navigate to the storage account list
+        * Click [+/Add] and enter values for the following fields:
+            * Subscription: Your preferred enterprise subscription for Cobalt
+            * Resource group: Create new (ex. cobalt-devint-admin-rg)
+            * Storage account name: (ex. cobalttfstates)
+        * Click [Review+Create] then [Create]
+        * Once deployment for storage account is completed, go to the resource and visit the Blobs sub-menu
+        * Click [+Container] then create a container name (ex. az-hw-remote-state-container ) with private access
+
+3. Configure Azure Devops pipeline using Azure resource values from previous steps
+
+    * Add the Azure Subscription being used for Cobalt as a *Service Connection*
+        * Return to your Azure DevOps subscription
+        * Find and select the Project Settings tab at the bottom of the screen
+        * Under the Pipelines menu select Service Connections
+        * From the Service Connections menu, select [+New Service Connection]
+        * Choose Azure Resource Manager from the dropdown and enter the Azure Subscription being used for Cobalt
+
+    * Configure *Infrastructure Pipeline Variables*
```

Comment:
Not clear what names should be used for variable groups. Should probably expand this section to explain the purpose of the variable groups (global and environment-specific), how it ties to the pipeline conventions, etc.